spec/controllers/api/v1/changes_controller_spec.rb

```diff
@@ -0,0 +1,118 @@
+
+require 'rails_helper'
+# Create
+RSpec.describe "Api::V1::Changes", type: :request do
+  describe "POST /api/v1/changes" do
+    let(:valid_params) do
+      {
+        user_id: 1,
```

Comment:
why is there no test case with the address?

Reply:
initially i focused on the 'name' field to verify the core functionality of the changes nonetheless i manually tested with both to ensure it was ok